testflo/main.py

```diff
@@ -131,7 +131,9 @@ def dir_exclude(d):
                                     dir_exclude=dir_exclude)
         benchmark_file = open(options.benchmarkfile, 'a')
     else:
-        discoverer = TestDiscoverer(dir_exclude=dir_exclude)
+        discoverer = TestDiscoverer(dir_exclude=dir_exclude,
+                                    func_pattern=six.text_type(options.test_glob),
```

Comment:
the func_pattern and the module_pattern won't necessarily always have the same prefix will they?